posthog/api/utils.py

```diff
@@ -424,7 +424,11 @@ def get_pk_or_uuid(queryset: QuerySet, key: Union[int, str]) -> QuerySet:
         UUID(key)  # type: ignore
         return queryset.filter(uuid=key)
     except ValueError:
+        pass
+    try:
         return queryset.filter(pk=key)
```

Comment:
Should we have a test here? They were passing a UUID to an endpoint that expected an integer...